llvm/test/CodeGen/RISCV/GlobalISel/legalizer/bswap32.mir

```diff
@@ -0,0 +1,180 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 3
+# RUN: llc -mtriple=riscv32 -run-pass=legalizer %s -o - | FileCheck %s
+
+---
+name:            bswap_i8
```

Comment:
How was this test created? i8 bswap isn't allowed in IR. The number of bytes must be even.

Reply:
I based off of https://github.com/llvm/llvm-project/pull/67577

Reply:
Will remove i8 test.